frontend/src/views/dashboard/DashboardSidebar.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styled from 'styled-components';
+import { DashboardTab } from './Dashboard';
+import { NoMarginSubtitleText } from '../../components/core/Text';
+import { PrimaryButtonLink, TextLink } from '../../components/core/Link';
+
+type DashboardSidebarProps = {
+  tab: DashboardTab,
+  onClick: (tab: DashboardTab) => void,
+};
+
+type TabItemProps = {
+  active: boolean,
+  onClick: (tab: DashboardTab) => void,
+  enabled?: boolean,
+};
+
+const Content = styled.div`
+  position: relative;
+  width: 300px;
```

Comment:
Sounds good - will prob stick with fixed width for now to avoid having to deal with odd proportions, but that's good to know if we wanted to refactor the design in the future at any point